data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Надо", "Nadeau" ],
+      [ "Ын", "Ng" ],
+      [ "Невилл", "Neville" ],
+      [ "Уикс", "Weeks" ],
+      [ "Нойманн", "Neumann" ],
+      [ "Барроуз", "Burrows" ],
+      [ "Ньюджент", "Nugent" ],
+      [ "Нэйлор", "Naylor" ],
+      [ "Нейпир", "Napier" ],
+      [ "О'Брайен", "O'Brien" ],
+      [ "О'Коннор", "O'Connor" ],
+      [ "Обриен", "Obrien" ],
+      [ "Одоннелл", "Odonnell" ],
+      [ "Лэйк", "Lake" ],
+      [ "Оукли", "Oakley" ],
+      [ "Оконнелл", "Oconnell" ],
+      [ "Олдем", "Oldham" ],
+      [ "Олири", "Oleary" ],
+      [ "Олстон", "Alston" ],
+      [ "Ольгин", "Holguin" ],
+      [ "Омалли", "Omalley" ],
+      [ "Онил", "Oneal" ],
+      [ "Онил", "Oneil" ],
+      [ "Ордонес", "Ordonez" ],
+      [ "Орейли", "Oreilly" ],
+      [ "Орурк", "Orourke" ],
+      [ "Марк", "Mark" ],
+      [ "Оу", "Oh" ],
+      [ "Хант", "Hunt" ],
+      [ "Хантер", "Hunter" ],
+      [ "Пикок", "Peacock" ],
+      [ "Паджет", "Padgett" ],
+      [ "Пак", "Pack" ],
+      [ "Коутс", "Coats" ],
+      [ "Пэрис", "Paris" ],
+      [ "Барбер", "Barber" ],
+      [ "Парэм", "Parham" ],
+      [ "Патино", "Patino" ],
+      [ "Пауерс", "Powers" ],
+      [ "Пате", "Pate" ],
+      [ "Пейн", "Payne" ],
+      [ "Пейнтер", "Painter" ],
+      [ "Пенс", "Pence" ],
+      [ "Стамп", "Stump" ],
+      [ "Эш", "Ash" ],
+      [ "Пералес", "Perales" ],
+      [ "Кросс", "Cross" ],
+      [ "Пеппер", "Pepper" ],
+      [ "Перселл", "Purcell" ],
+      [ "Сэндс", "Sands" ],
+      [ "Пиментел", "Pimentel" ],
+      [ "Пиплс", "Peoples" ],
+      [ "Пирес", "Pires" ],
+      [ "Роуч", "Roach" ],
+      [ "Вейр", "Weir" ],
+      [ "Карпентер", "Carpenter" ],
+      [ "Уэзерс", "Weathers" ],
+      [ "Ринг", "Ring" ],
+      [ "Филд", "Field" ],
+      [ "Баттс", "Butts" ],
+      [ "Портер", "Porter" ],
+      [ "Пратт", "Pratt" ],
+      [ "Принс", "Prince" ],
+      [ "Париш", "Parish" ],
+      [ "Форман", "Foreman" ],
+      [ "Рид", "Read" ],
+      [ "Пруэтт", "Pruett" ],
+      [ "Берд", "Bird" ],
+      [ "Пуарье", "Poirier" ],
+      [ "Крэйвен", "Craven" ],
+      [ "Бланк", "Blank" ],
+      [ "Пэрротт", "Parrott" ],
+      [ "Пэтти", "Petty" ],
+      [ "Бутчер", "Butcher" ],
+      [ "Ракер", "Rucker" ],
+      [ "Шелл", "Shell" ],
+      [ "Ралстон", "Ralston" ],
+      [ "Эрли", "Early" ],
+      [ "Ранхель", "Rangel" ],
+      [ "Раст", "Rust" ],
+      [ "Комбс", "Combs" ],
+      [ "Ратледж", "Rutledge" ],
+      [ "Редмен", "Redman" ],
+      [ "Рентерия", "Renteria" ],
+      [ "Риверс", "Rivers" ],
+      [ "Роарк", "Roark" ],
+      [ "Робисон", "Robison" ],
+      [ "Хорн", "Horn" ],
+      [ "Родос", "Rhodes" ],
+      [ "Роквелл", "Rockwell" ],
+      [ "Ролз", "Rawls" ],
+      [ "Ропер", "Roper" ],
+      [ "Роуленд", "Rowland" ],
+      [ "Роша", "Rocha" ],
+      [ "Гроув", "Grove" ],
+      [ "Гровс", "Groves" ],
+      [ "Роэ", "Roe" ],
+      [ "Роял", "Royal" ],
+      [ "Рупп", "Rupp" ],
```

Comment:
This is a name of German origin, where it would indeed be transcribed as `Рупп`. However, given that the game is set in the US and that there are quite a few Americans with that name ([`Дебра Джо Рапп`](https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%BF%D0%BF,_%D0%94%D0%B5%D0%B1%D1%80%D0%B0_%D0%94%D0%B6%D0%BE) and [`Гален Рапп`](https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%BF%D0%BF,_%D0%93%D0%B0%D0%BB%D0%B5%D0%BD) being the ones I could verify), it should probably be `Рапп`.